torch/_dynamo/variables/dicts.py

```diff
@@ -241,6 +241,12 @@ def call_method(
         elif name == "pop" and arg_hashable and self.mutable_local:
             tx.output.side_effects.mutation(self)
             return self.items.pop(Hashable(args[0]))
+        elif name == "clear":
+            assert len(args) == 0
+            assert len(kwargs) == 0
+            keys_vt = [key.vt for key in self.items]
+            for key_vt in keys_vt:
+                self.call_method(tx, "pop", [key_vt], {})
```

Comment:
Why not just:
```
tx.output.side_effects.mutation(self)
return self.items.clear()
```
?

Reply:
Yeah, that would work and have better performance. I was hesitating between performance and re-using the code path. I can change to the suggested modification.